trunk/SUAVE/Input_Output/VTK/save_fuselage_vtk.py

```diff
@@ -0,0 +1,127 @@
+## @ingroup Input_Output-VTK
+# save_fuselage_vtk.py
+# 
+# Created:    Jun 2021, R. Erhard
+# Modified: 
+#  
+
```

Comment:
Can you put some comments blocks to divide up this script? Like that say import, or in the case of a function the name of the function? It helps to navigate large scripts.

Reply:
added comment blocks